docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,201 @@
+# Styling UI components with XAML
+Styling of components in NUI is possible in two ways:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
+
+## ViewStyle theme
+The basic styling of NUI components is based on the `ViewStyle` class and its derivatives. `ViewStyle` defines attributes and stores their values. 
+
+The following is a list of the most common attributes:  
+- name
+- state 
+- subState
+- position
+- scale
+- orientation
+- padding 
+
+You can find all the attributes and their details on [ViewStyle attributes](https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs) page.
+
+In addition, styles can also be defined for the components with following states:
+| State              | Description                                                                                                                                          |
+|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Normal`           | Specifies the default style.                                                                                                                         |
+| `Focused`          | Specifies the state where all the keystrokes and events are recieved by the widget. There can only be one widget in the active window at a time in this state. |
+| `Disabled`         | Specifies the state where the widget is disabled. All other events are ignored in this state.                                                          |
+| `Selected`         | Specifies the state where a widget is selected.                                                                                                      |
+| `Pressed`          |  Specifies the state where the widget receives touch event.                                                                                                                                                   |
+| `DisabledFocused`  | Specifies the state that is similar to the `Focused` state, but there is no possibility to change the widget state.                                                                        |
+| `SelectedFocused`  | Specifies the state where the widget is checked and all events are received.                                                                                            |
+| `DisabledSelected` | Specifies the combination of disabled and selected states.                                                                                                       |
```

Comment:
Minor Question. 
It is said the ViewStyle is for the NUI components, so the ViewStyle would be applied to the components.
In this table the components is called "widget". 
=> Is it Ok not to name it "view", but to "widget"?

Question. 
[In this comment](https://github.com/Samsung/tizen-docs/pull/1386#discussion_r561746700) most updates are about adding "the".
"the" is also added to every "state", but isn't "a" more accurate to each "state"?


Specifies the state where the widget is disabled. All other events are ignored in this state.
=> In this state, every events are ignored not excepting any. So I think removing **other** would be better.
Specifies the state where the widget is disabled. All  events are ignored in this state.

Question.
Specifies **the** state where **a** widget is selected.
=> Till here "the" is used to the word "widget" in other sentences. At here why **a** is used? Is there any reason for this difference?


Also another question about "where" in below sentences: 
In the middle of the sentences, "where" is added. 
However according to the description, as this table is about the states of the components, isn't "whether" or "that" more proper? 
 - Specifies the state **that** a widget is selected.
 - Specifies the state **that** the widget receives touch event.
 - Specifies the state **that** the widget is checked and all events are received.